pennylane/__init__.py

```diff
@@ -56,7 +56,7 @@
     mutual_info,
 )
 from pennylane.ops import *
-from pennylane.ops import adjoint, ctrl
+from pennylane.ops import adjoint, ctrl, op_sum
```

Comment:
How come it is imported as `op_sum`? Due to namespacing, since nothing in this file uses the intrinsic python `sum`, I would think it would be okay to import it as `sum` (allowing for `qml.sum`)?

Reply:
I think I had initially swapped it because I was running into an error due to the builtin python sum method.

The only think we would need to worry about is the case where a user tries something like: 

```python 
from pennylane import * 

# ... some pennylane code ...

lst = [1, 2, 3]
x = sum(lst)

# ... some more code ...
```

Where they unknowingly overwrite this function. If we keep it `op_sum`, then its clear that this is a different type of sum method. Also since we will be overwriting `__add__` in the long term, we expect to users to use that anyways, instead of `sum`.